runtime/queries/glsl/indents.scm

```diff
@@ -0,0 +1 @@
+; inherits: c
```

Comment:
We use `indents.toml` instead: https://github.com/helix-editor/helix/issues/114

Reply:
@kirawi  I added `indents.toml` based on https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/c/indents.scm